connectors/kafka/src/test/java/forklift/connectors/KafkaControllerTests.java

```diff
@@ -52,14 +53,27 @@ public void teardown() throws InterruptedException {
     }
 
     @Test
-    public void addTopicTrueTest() {
+    public void addTopicTrueTest() throws InterruptedException {
+        this.controller.start();
         String topic1 = "topic1";
         boolean added = this.controller.addTopic(topic1);
         assertEquals(true, added);
     }
 
     @Test
-    public void addTopicFalseTest() {
+    public void addingTheSameTopicRepeatedlyDoesntLock() throws InterruptedException {
+
+        this.controller.start();
+        String topic1 = "topic1";
+        for(int i = 0; i < 10; i++){
+            this.controller.addTopic(topic1);
+        }
+        assertTrue(true);
```

Comment:
Seems a bit redundant; if you want to clarify this test, an comment would be more expressive